.env.example

```diff
@@ -33,6 +33,12 @@ PUSHER_SECRET=
 
 FACEBOOK_CLIENT_ID=1825537217668365
```

Comment:
[rank SSS] please remove all keys on env.example. We just put key name only and leave it as blank